index.html

```diff
@@ -47,32 +47,173 @@ <h1>Gottfrids Munkfabrik</h1>
     <!-- Under denna kommentar lägger vi till 'erbjudande' och alla munkar -->
     <main>
 
-    <!-- Sökfältet-->
-      <section>
-        <input type="Search" placeholder="Search">
-        <a href="#"><i class="fa-solid fa-magnifying-glass"></i></a>
-      </section>
+        <!-- Sökfältet-->
+        <section>
+            <input type="Search" placeholder="Search">
+            <a href="#"><i class="fa-solid fa-magnifying-glass"></i></a>
+        </section>
 
-    <!--Kampanj-->
-      <section>Kampanj</section>
-      <div>
-          <h2>Köp 15 munkar och få gratis frakt!</h2>
-      </div>
+        <!--Kampanj-->
+        <section>Kampanj</section>
+        <div>
+            <h2>Köp 15 munkar och få gratis frakt!</h2>
+        </div>
         <!--Denna div syns när en tidsaktiverad kampanj är aktiv-->
-      <div class="limited-time-campaign">
-        <h2><!--Texten här varierar beroende på vilken rebatt som gäller då, jag sätter visibility: hidden tills dess--></h2>
-      </div>
-        
+        <div class="limited-time-campaign">
+            <h2><!--Texten här varierar beroende på vilken rebatt som gäller då, jag sätter visibility: hidden tills dess--></h2>
+        </div>
 
-    <!--Filtrera och sortera-->
-      <section>
-        <span>Filtrera och sortera</span>
-        <a href="#"><!--plats för ikon--></a>
-      </section>
-      
-    <!--Munkar-->
-      <section></section>
-      
+
+        <!--Filtrera och sortera-->
+        <section>
+            <span>Filtrera och sortera</span>
+            <a href="#"><!--plats för ikon--></a>
+        </section>
+
+        <!--Munkar-->
+        <section></section>
```

Comment:
Jag ser att den här är oanvänd just nu. Borde vi ta bort den eller stoppa in munkarna i den?